blog/2024-01-31-january-releases.mdx

```diff
@@ -0,0 +1,246 @@
+---
+title: "January 2024 Releases"
+slug: "2024-01-31-january-releases"
+authors: [kenodegard]
+tags: [announcement, conda, conda-build, conda-libmamba-solver, conda-index]
+description: |
+  Conda 24.1.0/.1/.2, conda-build 24.1.0/.1/.2, conda-libmamba-solver 24.1.0, and conda-index 0.4.0 have been released! 🎉
+image: img/blog/2024-01-31-january-releases/banner.jpg
+---
+
+The January 2024 releases included updates to four conda community projects: conda, conda-build, conda-libmamba-solver, and conda-index! 🎉 Each of these have been released to both `main` and `conda-forge`.
+
+## Changes in Conda [24.1.0](https://github.com/conda/conda/releases/tag/24.1.0)/[24.1.1](https://github.com/conda/conda/releases/tag/24.1.1)/[24.1.2](https://github.com/conda/conda/releases/tag/24.1.2)
+
+To update conda, run:
+
+```bash
+conda install -n base conda=24.1.2
+```
+
+### ✨ What's New? ✨
+
+- Verify signatures on to-be-installed packages instead of on all packages.
+- Add new `pre-solves` and `post-solves` plugin hooks.
+- Add support for Python 3.12.
```

Comment:
```suggestion
- Signature verification on to-be-installed packages instead of on all packages.
- New `pre-solves` and `post-solves` plugin hooks.
- Python 3.12 support.
```